hs_core/page_processors.py

```diff
@@ -186,7 +186,9 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                    'bag_url': bag_url,
                    'show_content_files': show_content_files,
                    'discoverable': discoverable,
-                   'is_mine': content_model.is_mine
+                   'is_mine': content_model.is_mine,
+                   'res_id': content_model.short_id,
+                   'extra_matadata_dict': content_model.extra_metadata
```

Comment:
@pkdash I believe it is the same for extra_metadata. There is no need to pass extra_metadata_dict context to template, instead you can directly use ```cm.extra_metadata``` to retrieve extra metadata dict in genericresource.html template.